worker/include/RTC/IceServer.hpp

```diff
@@ -17,9 +18,14 @@ namespace RTC
 			NEW = 1,
 			CONNECTED,
 			COMPLETED,
-			DISCONNECTED
+			DISCONNECTED,
+			CLOSED
```

Comment:
Added `CLOSED` for consistency with Node and Rust. This can probably be removed though.

I'll review this.

Reply:
When is CLOSED needed?

Reply:
It is not. But it's present in TS and RUST. I'll create a TODO to remove them in this PR.

Reply:
Done.